SoftLayer/managers/hardware.py

```diff
@@ -697,8 +697,11 @@ def get_bandwidth_allocation(self, instance_id):
         a_mask = "mask[allocation[amount]]"
         allotment = self.client.call('Hardware_Server', 'getBandwidthAllotmentDetail', id=instance_id, mask=a_mask)
         u_mask = "mask[amountIn,amountOut,type]"
-        useage = self.client.call('Hardware_Server', 'getBillingCycleBandwidthUsage', id=instance_id, mask=u_mask)
-        return {'allotment': allotment.get('allocation'), 'useage': useage}
+        usage = self.client.call('Hardware_Server', 'getBillingCycleBandwidthUsage', id=instance_id, mask=u_mask)
+        allotment_bandwidth = None
+        if allotment is not "":
```

Comment:
just do `if allotment` here. The tricky bit is that the XMLRPC endpoint likes to return `None`, and REST will return `''`. Luckily both will return False in python.

Reply:
that will also fix the Tox error about comparing to a literal.

Reply:
Try using with this  
`if myString in (None, '')`

Reply:
Done.